docs/4-user-guide/2-zarf-packages/2-zarf-components.md

```diff
@@ -43,10 +43,13 @@ components:
      name: flux-v1.0.0
 ```
 
-> Note: When importing a component, Zarf will copy all of the values from the original component except for the `required` key. In addition, while Zarf will copy the values, you have the ability to override the value for the `description` key.
+:::note
+When importing a component, Zarf will copy all of the values from the original component except for the `required` key. In addition, while Zarf will copy the values, you have the ability to override the value for the `description` key.
```

Comment:
```suggestion

When importing a component, Zarf will merge in all of the values from that component except for the `required` key - this allows the component you are importing from to specify override values that will be merged together with the base values from the imported component (i.e. adding additional `manifests`, `images`, or `repos`)
```

Reply:
This is maybe a clearer way to describe this feature - While the old text hinted at it, this helps expand on what it might be used for (we also at some point should update that example because it doesn't show that at all...)